ext/intl/timezone/timezone.stub.php

```diff
@@ -112,6 +112,14 @@ public function getErrorMessage(): string|false {}
      */
     public static function getGMT(): IntlTimeZone {}
 
+#if U_ICU_VERSION_MAJOR_NUM >= 74
+    /**
+     * @tentative-return-type
```

Comment:
feel free to use a real return type because there is no BC impact of new methods (besides accidental collision iwth similarly named userland methods)